target/linux/ramips/dts/ApexeraWiBot.dts

```diff
@@ -0,0 +1,170 @@
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "apexera-wibot", "ralink,mt7620a-soc";
+	model = "Apexera WiBot";
+
+	chosen {
+		bootargs = "console=ttyS1,57600";
+        };
+
+	gpio-leds {
+		compatible = "gpio-leds";
+
+		wan {
+			label = "wibot:blue:wan";
+			gpios = <&gpio1 5 1>;
```

Comment:
Use the GPIO_ACTIVE_LOW and GPIO_ACTIVE_HIGH macros instead of 1 and 0. Check the existing dts files for reference.